docs/notebooks/expected_improvement.pct.py

```diff
@@ -176,9 +176,7 @@ def build_model(data):
 # We can also inspect the model hyperparameters, and use the history to see how the length scales evolved over iterations. Note the history is saved at the *start* of each step, and as such never includes the final result, so we'll add that ourselves.
 
 # %%
-gpflow.utilities.print_summary(
-    result.try_get_final_models()[OBJECTIVE].model  # type: ignore
-)
+gpflow.utilities.print_summary(result.try_get_final_models()[OBJECTIVE].model)  # type: ignore
```

Comment:
(not part of your change but curious why the type: ignore is necessary. might investigate...)